modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -149,21 +149,65 @@ const opMakeRowHeader = function (grid: Structs.RowCells[], detail: Structs.Deta
   return bundle(newGrid, detail.row, detail.column);
 };
 
-const opMakeColumnHeader = function (grid: Structs.RowCells[], detail: Structs.DetailExt, comparator: CompElm, genWrappers: GeneratorsTransform) {
-  const newGrid = TransformOperations.replaceColumn(grid, detail.column, comparator, genWrappers.replaceOrInit);
+// Function added to match opMakeColumnsHeader, but not used anywhere.
```

Comment:
This isn't true, it's available in the public API. While it's not currently used anywhere, it very well could be and then this comment would be very misleading. So I'm going to ask that you remove this please, as it's the type of thing that while accurate now will degrade over time.

- [ ] Remove the 2 comments like this

Reply:
Ah, I mentioned to Hannes about adding the comments tbh. At the time we were discussing how some (at least one?) method in the current API isn't used anywhere, which was a little confusing. I get what you mean about it degrading though... Hm.